runconfig/config.go

```diff
@@ -30,6 +31,7 @@ type Config struct {
 	Cmd             *stringutils.StrSlice // Command to run when starting the container
 	Image           string                // Name of the image as it was passed by the operator (eg. could be symbolic)
 	Volumes         map[string]struct{}   // List of volumes (mounts) used for the container
+	CLIVolumeSpecs  map[string]struct{}   // CLI-passed list of volumes (mounts) used for the container
```

Comment:
How about `RawMounts` or `RawVolumes` or `Raw[Something]` ? I don't like `CLI` and I don't like `Specs`. /cc @calavera @cpuguy83